Source/SpatialGDK/Public/SpatialInterop.h

```diff
@@ -93,10 +94,12 @@ using FOutgoingPendingArrayUpdateMap = TMap<const UObject*, FChannelToHandleToOP
 
 using FResolvedObjects = TArray<TPair<UObject*, const improbable::unreal::UnrealObjectRef>>;
 
-// Helper function to write incoming replicated property data to an object.
-FORCEINLINE void ApplyIncomingReplicatedPropertyUpdate(const FRepHandleData& RepHandleData, UObject* Object, const void* Value, TSet<UProperty*>& RepNotifies)
+// Helper function to write incoming replicated property data to an object
+// TargetObject is the actor or actor component that owns the replicated property and its RepNotifies, NOT the root actor channel
+// to-do: unify how replicated properties and handover are handled
```

Comment:
I'm confused, isn't this function dealing with them in the same way? How do you want to unify them more? 
And if you add a todo, please add a tracking Jira ticket.

Reply:
Ahh don't mind me, I see they have different property updaters :)
Please add a Jira ticket though.

Reply:
done